### PR TITLE
Gets the homepage for the site working including a header. Fixes #11.

### LINKED ISSUE #11: Create home page
**Done when:**
Going to the root of the project is a page other than the default rails server page and there are controller tests for it

### PR DESCRIPTION
**Pull request by avatousiou**

### CODE REVIEW

File: spec/controllers/home_controller_spec.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+describe HomeController do
+  
+  describe "GET home" do
+    it "gets the users" do
+      get :home
+      expect(assigns(:users)).to_not be nil
+    end
+  end
+end
```
Comment (afred) [inline spec/controllers/home_controller_spec.rb:5]: Doesn't look like you are actually using `@users` anywhere except in this test, right?

**Comment by avatousiou** [comment]
Yeah, I wasn't sure what else to test at the moment since there isn't anything else

**Comment by avatousiou** [comment]
Hey @afred I removed that test so should be good now

**Comment by afred** [comment]
@avatousiou ok, I'll merge. The point I was trying to make though, was that the piece of code that the test was testing is not really needed. I.e. I dont' think you [need this](https://github.com/WGBH/stock_sales/pull/12/files#diff-14dfef9f01460f69f175a8e6bd7d143dR4) for anything. If you _do_ need that, then keeping the test around is fine. If you _don't_ need that, then get rid of that piece along with any tests that would fail as a result of removing the unneeded code. Make sense?

**Comment by avatousiou** [comment]
Oh, yeah forgot about that bit. I'll remove it with the next ticket

**Pull request merged**